### PR TITLE
made changes to improve Performance, SEO, Accessibility of the site

### PR DESCRIPTION
**Pull request by reveurguy**
Signed-off-by: Priyanshu Singh <reveur.guy99@gmail.com>

Made changes to improve Performance, SEO, Accessibility of the website.

## Before: 
Screenshot 2022-08-12 at 5 12 33 PM

## After: 
Screenshot 2022-08-12 at 5 12 11 PM

The performance should have been improved a little bit as well, my slow and fluctuating internet connection can be the reason behind the difference in performance scores, but on testing locally, the performance was around 90-91. 

The performance can be further improved currently as well as best practices and seo.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: layouts/partials/logo-section.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
       {{ $logo := printf "img/logos/%s" .logo | relURL }}
       <div class="column is-one-quarter">
         <a href="{{ .url }}" target="_blank">
-          <img src="{{ $logo }}">
+          	<img src="{{ $logo }}" alt=" Company icons">
```
Comment (chalin) [inline layouts/partials/logo-section.html:20]: This partial is used for more than "Company icons".

```suggestion
          	<img src="{{ $logo }}" alt="Logo">
```

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <div class="container">
     <div class="columns is-vcentered is-variable is-8">
       <div class="column is-one-third">
-        <img class="is-footer-logo" src="{{ $logo }}" alt="{{ $title }} footer logo">
+        <img class="is-footer-logo" src="{{ $logo }}" alt="{{ $title }} footer logo" alt="Logo">
```
Comment (chalin) [inline layouts/partials/footer.html:11]:
```suggestion
        <img class="is-footer-logo" src="{{ $logo }}" alt="{{ $title }} footer logo" alt="Keda logo">
```

File: layouts/partials/css.html [hunk 3]
```diff
@@ -12,4 +12,4 @@
 {{ $prodCss      := $css | fingerprint }}
 <link rel="stylesheet" href="{{ $prodCss.RelPermalink }}" integrity="{{ $prodCss.Data.Integrity }}">
 {{ end }}
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
\ No newline at end of file
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2.6.3/dist/cdn/docsearch.min.css" />
```
Comment (chalin) [inline layouts/partials/css.html:15]: Are you claiming that this affects accessibility ratings?
Revert this change, `@2` is enough to get `@2.6.3`.

File: config.toml [hunk 4]
```diff
@@ -2,6 +2,7 @@ baseURL = "https://keda.sh"
 languageCode = "en-us"
 title = "KEDA"
 enableGitInfo = true
+enableRobotsTXT = true
```
Comment (chalin) [inline config.toml:5]: Are you claiming that this affects accessibility ratings? Revert this change.

File: layouts/partials/logo-section.html (hunk 1 above)
Comment (chalin) [reply layouts/partials/logo-section.html:20]: Actually, we can probably do better than this by extracting the domain name from the `.url` and using that as `alt` text.

File: layouts/partials/meta.html [hunk 5]
```diff
@@ -9,6 +9,9 @@
 {{ $locale      := site.Params.locale }}
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta
+  name="description"
+  content="KEDA is a Kubernetes-based Event Driven Autoscaler. With KEDA, you can drive the scaling of any container in Kubernetes based on the number of events needing to be processed.">
```
Comment (chalin) [inline layouts/partials/meta.html:14]: Good of you to point out that pages need a `<meta name="description">`, but this isn't the way to do it.
Revert this change.

Comment (chalin) [reply layouts/partials/meta.html:14]: This will be tracked via:

- #894

File: layouts/partials/navbar.html [hunk 6]
```diff
@@ -12,7 +12,7 @@
     <div class="navbar-brand is-left-padded" style="align-items: end;">
       <a class="navbar-item" href="{{ $home }}">
         {{ $url := printf "img/logos/%s" $logo | relURL }}
-        <img src="{{ $url }}" style="transform: scale(2);">
+        <img src="{{ $url }}" style="transform: scale(2);" height="28" width="67" alt="Logo">
```
Comment (chalin) [inline layouts/partials/navbar.html:15]:
```suggestion
        <img src="{{ $url }}" style="transform: scale(2);" height="28" width="66" alt="Keda logo">
```

**Review by chalin — CHANGES_REQUESTED** [review]
Overall, nice improvements, thanks. 

See inline comments, there are a handful of changes that need to be **reverted** and/or changed.

File: config.toml (hunk 4 above)
Comment (reveurguy) [reply config.toml:5]: Hi chalin, this was for improving SEO to appear in search results and search engine indexing.

**Comment by reveurguy** [comment]
thank you chalin for the suggestions, i'll work on them.

File: layouts/partials/footer.html (hunk 2 above)
Comment (reveurguy) [reply layouts/partials/footer.html:11]: there are 2 alt tags here, shouldn't i add the keda logo one and remove the other

File: layouts/partials/css.html (hunk 3 above)
Comment (reveurguy) [reply layouts/partials/css.html:15]: there was some kind of error being shown in the console and when i updated to the latest version it went away, but i wonder that might have been due to some network issue because i am not getting that error again.
I'll revert this.

File: layouts/partials/footer.html (hunk 2 above)
Comment (chalin) [reply layouts/partials/footer.html:11]: Yes, thanks

**Comment by reveurguy** [comment]
i have made the changes leaving the config.toml, footer.html and for the logo-section i have currently added Logo. I'll make the changes to the remaining files, as soon as the changes are finalised for them.

File: config.toml (hunk 4 above)
Comment (reveurguy) [reply config.toml:5]: should i keep this or remove it?

Comment (chalin) [reply config.toml:5]: Drop it for this PR. I'm willing to be convinced otherwise, but as far as I know this buys us nothing.

Comment (reveurguy) [reply config.toml:5]: ok i'll revert this

**Comment by reveurguy** [comment]
I have made the changes as discussed in the footer file and config file.

**Comment by reveurguy** [comment]
Hii, any update on this PR? Do let me know if any changes are required.

**Comment by tomkerkhove** [comment]
@chalin @thisisobate Can you please take a look at this PR?

File: layouts/partials/footer.html [hunk 7]
```diff
@@ -8,7 +8,7 @@
   <div class="container">
     <div class="columns is-vcentered is-variable is-8">
       <div class="column is-one-third">
-        <img class="is-footer-logo" src="{{ $logo }}" alt="{{ $title }} footer logo">
+        <img class="is-footer-logo" src="{{ $logo }}" alt="Keda Logo">
```
Comment (tomkerkhove) [inline layouts/partials/footer.html:11]:
```suggestion
        <img class="is-footer-logo" src="{{ $logo }}" alt="KEDA Logo">
```

**Comment by reveurguy** [comment]
Any update on this?

**Review by reveurguy — COMMENTED** [review]
made changes

**Comment by tomkerkhove** [comment]
[@nate-double-u](https://github.com/nate-double-u) any update on this?

**Comment by tomkerkhove** [comment]
@nate-double-u any update on this?

**Review by nate-double-u — APPROVED** [review]
> @nate-double-u any update on this?

It looks like @chalin's feedback has been integrated except for a couple bits that can be done in a follow up PR.

/lgtm

**Comment by tomkerkhove** [comment]
Thanks @nate-double-u!
